### DISCUSSION TITLE
Is it possiable to clip a window and its titlebar inside a client rect?

### DISCUSSION THREAD

**Discussion by Shadowblitz16** [picoe/Eto#2157] [Q&A]
Is it possible to manually clip a window and its titlebar inside a client rect?

If so it might be a valid mdi work around.

**Comment by cwensley** [comment]
Nope.. Ideally, as I've mentioned before, is to use a Drawable with a PixelLayout to lay out different controls with "window" decorations.  Even Windows Forms MDI has [old-style window decorations](https://github.com/dotnet/winforms/issues/3691) because it hasn't been updated in years.

**Reply by Shadowblitz16** [reply]
wait what do you mean by window decorations? Can you render the native titlebar?
I thought the titlebar wasn't part of the rendering api?

**Reply by cwensley** [reply]
No, there is no way to render the native title bar (on any platform, otherwise Windows Forms would not have the aforementioned issue).  You'd need to render your own.
